BLSQ/iaso
# Ia 1798 add expandable item

Move the generic ExpandableItem out of Iaso, to bluesquare-components

Related JIRA tickets : IA-1978

## Self proofreading checklist


## Changes

- Replaced `ExpandableItem` with the bluesqure-components version
- Fixed a minor syntax error in Budget modals

## How to test

- Look for uses of the component in the code base. Go to pages that use it, nothing should have changed
- I spotted 3 uses:
    - In polio budget tab
    - In polio history log
    - In Deduplication, details. There's a bug when you expand the "Informations" section. It's unrelated and fixed in another (upcoming) deduplication PR

## Print screen / video


https://github.com/BLSQ/iaso/assets/38907762/71828c13-4d36-435e-9a0c-9082ae4d2293

https://github.com/BLSQ/iaso/assets/38907762/c00bc856-1413-4a3c-8f7d-3818c940a532


## Notes

Requires https://github.com/BLSQ/bluesquare-components/pull/119
